config/webpack.prod.js

```diff
@@ -11,6 +11,7 @@ module.exports = webpackMerge(commonConfig, {
     devtool: 'source-map',
 
     plugins: [
+        new webpack.optimize.CommonsChunkPlugin({ name: ['main', 'vendor', 'polyfills'], minChunks: Infinity }),
```

Comment:
Why was this causing problem? Since we have minChunks to be Infinity, it shouldn't move anything in a common chunk file.